docs/content/https/acme.md

```diff
@@ -8,6 +8,45 @@ You can configure Traefik to use an ACME provider (like Let's Encrypt) for autom
 !!! warning "Let's Encrypt and Rate Limiting"
     Note that Let's Encrypt API has [rate limiting](https://letsencrypt.org/docs/rate-limits).
 
+    Use Let's Encrypt staging server with the [`caServer`](#caserver) configuration directives
```

Comment:
```suggestion
    Use Let's Encrypt staging server with the [`caServer`](#caserver) configuration option
```

Reply:
Challenging this suggestion: looks fine, but how do we deal with the potential confusion with `tls.options` option? I initially wrote option as you suggests, but felt that `directives` was a mor non-ambiguous wording in this context (TLS). WDYT?

Reply:
We will keep `option` to be homogeneous with the other documentation